docs/api/README.md

```diff
@@ -47,7 +47,10 @@ It contains a coma-separated list of versions, e.g., `X-Notes-API-Versions: 0.2,
 
 ## Major API versions
 
-TODO Table
+| Version | Supported by app version | Remarks |
+|:-------:|--------------------------|---------|
+| **0.2** | since 1.0 (2015)         | *(deprecated)* |
+| **1**   | since 3.3 (April 2020)   | see [documentation](v1.md) |
```

Comment:
@korelstar Is it really `1` or `1.0`? I saw different spellings, it should be the same everywhere to avoid confusion.

Reply:
This table is about the major version, so it's `1`. (`0.2` is a special case, but this versioning approach is compatible to the "semver" approach [except that we don't have a patch version fro the API].) The major version determines the API's base path and all minor versions of the same major version are compatible to each other - if the points from the next section are realized on the client side.

Is this comprehensible? How could we improve the description?